packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -39,9 +39,49 @@ type UpdateOptions = {
   input: TUpdateInputSchema;
 };
 
+const verifyEmailSender = async (
+  email: string,
+  userId: number,
+  teamId: number | null,
+  prisma: PrismaClient
+) => {
+  const verifiedEmail = await prisma.verifiedEmail.findFirst({
+    where: {
+      email,
+      OR: [{ userId }, { teamId }],
+    },
+  });
+
+  if (!verifiedEmail) {
+    throw new TRPCError({ code: "NOT_FOUND", message: "Email not verified" });
+  }
+
+  if (teamId) {
+    if (!verifiedEmail.teamId) {
```

Comment:
If is team workflow but email was only verified for user, then add it to team as well.

Reply:
`verifyEmailSender` is only called when reminders are being scheduled which is not always the case

Reply:
`verifyEmailSender` is only called when new reminders are scheduled which is not always the case

Reply:
Not sure wdym by that. I called it everywhere we were using sendTo for EMAIL_ADDRESS action.

Reply:
We don't call `verifyEmailSender` when no reminders are scheduled. So for example you can do that: 

1. Create workflow with 'When event is booked' trigger 
2. Choose 'send Email to specific email address' 
3. Type a not verified email address 
4. Type random numbers as the verification code
5. Click save 

This now successfully saves the workflow without `verifyEmailSender` ever being called.

Reply:
Yeah should have been checking before submitting form fixed this.